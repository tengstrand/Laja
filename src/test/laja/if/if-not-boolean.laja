#write "target/test-output/if-not-boolean.out"
  #set (flag = false)
  #if ( ! false)
!false is true
  #end
  #if (!flag)
!flag is true
  #else
!flag is false
  #end
#end