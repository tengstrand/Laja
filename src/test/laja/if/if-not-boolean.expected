!false is true
!flag is true
